File: app/src/main/java/org/flyontime/android/model/data/FlyOnTime/DateModel.java
package org.flyontime.android.model.data.FlyOnTime;

import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * Created by jossi on 18.06.2017.
 */

public class DateModel extends DashboardModelInterface {

    private Date date;

    public DateModel(Date date) {
        this.date = date;
    }

    public Date getDate() {
        return date;
    }

    public String getShortenedDateString() {
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd MMM hh:mm");
        return simpleDateFormat.format(date);
    }
}

File: app/src/main/java/org/flyontime/android/model/data/FlyOnTime/DateModel.kt
package org.flyontime.android.model.data.FlyOnTime

import android.annotation.SuppressLint
import java.text.SimpleDateFormat
import java.util.*

/**
 * Created by jossi on 18.06.2017.
 */

data class DateModel(val date: Date) : DashboardModelInterface() {

    val shortenedDateString: String
        get() {
            @SuppressLint("SimpleDateFormat")
            val simpleDateFormat = SimpleDateFormat("dd MMM hh:mm")
            return simpleDateFormat.format(date)
        }
}
